scss/civicase.scss

```diff
@@ -7,6 +7,8 @@
 
 @import 'SCSSROOT/org.civicrm.shoreditch/scss/bootstrap/overrides/variables';
 @import 'SCSSROOT/org.civicrm.shoreditch/base/scss/overrides/variables';
+@import 'SCSSROOT/org.civicrm.shoreditch/base/scss/vendor/bootstrap/variables';
```

Comment:
wouldn't placing the vendor variables after the variables override on line 8 replace our overrides with the vendors original values?

Reply:
No it wont actually. Here is the explanation https://stackoverflow.com/a/36968517.